src/consensus/pbft/libbyz/Replica.h

```diff
@@ -497,6 +499,10 @@ class Replica : public Node, public IMessageReceiveBase
   // used to register a callback for a client proxy to collect replies sent to
   // this replica
 
+  GovernanceRequestTracking gov_req_track;
+  // used to track the last series of pre-prepares that had requests that
+  // affected the goverenance of the service
```

Comment:
governance

Reply:
thanks! my spell checking adding is really failing me lately. Any chance you have a good one?